notebooks/.gitignore

```diff
@@ -0,0 +1,31 @@
+# Files generated by convert.py
+# -----------------------------
+exec_*.ipynb
+*.html
+*.fits
+*.coo
+*.match
+*.log
+*.png
```

Comment:
I think we should remove .pngs and .jpeg from ignored as there are many present in the repo and its common for them to be included in markup sections of the notebook.

```
./assets/stsci_pri_combo_mark_horizonal_white_bkgd.png
./notebooks/MAST/K2/K2_FFI/kepler_focal_plane_layout_channels_color.png
./notebooks/MAST/Kepler/Kepler_FFI/ffi_tic_plot.png
./notebooks/MAST/Kepler/Kepler_TPF/tpf_fluxplot_28-29.png
./notebooks/MAST/Kepler/Kepler_TPF/tpf_fluxplot0.png
./notebooks/MAST/Kepler/Kepler_Lightcurve/light_curve_tres2.png
./notebooks/MAST/HSC/HSC_TAP/smc_colormag.png
./notebooks/MAST/PanSTARRS/PS1_DR2_TAP/stsci_pri_combo_mark_horizonal_white_bkgd.png
./notebooks/DrizzlePac/sky_matching/MDRIZSKY_Values.png
```

Reply:
I just pushed a new commit that does this, but I will say that removing `*.png` from the list means `convert.py` creates about 100 untracked pngs that appear in `git status` now. 🙁 

This might be too much trouble, but I was thinking that we _could_ move all of the supplemental pngs you list above into `assets` subdirectories within the notebooks folder? So then we could just add `!**/assets` to the `.gitignore` and they would be ignored...

Reply:
Ah, apologies, I wasn't aware that running the notebooks also produced pngs. I suppose a simple `find . -name "*.png"` to find all current present included .pngs can be used and then add them to the gitignore with ! may be the way to go. Most of the pngs belong to the individual notebooks besides the one in assets.

Reply:
Okay, that sounds like a good fix to me! Just pushed a commit to do that.

Reply:
Do you think we should add the entire assets folder?
`!assets/*`, that way anything added to that is automatically excluded. I believe the idea for this folder was for global assets that all notebooks may use.

Reply:
Good point - but since this `.gitgnore` file is inside the `notebooks/` subdirectory, nothing in it would be applied to the `assets/` directory anyway. It only applies for things in `notebooks/notebooks/` and the subdirectories in there.